src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -63,6 +63,13 @@
  * of the image.
  *
  * <p>
+ * If the image source parameter to a constructor or method is non-null,
+ * but does not reference valid accessible image data,
+ * no exceptions will be thrown but the image will be unset,
```

Comment:
"will behave as if unset" ?

Reply:
@prrace Any objection to my suggestion of statement? I guess that is succinct and describe what's the end result!!
`no exceptions will be thrown but the image will behave as if unset/uninitialized/null` doesn't sound right in my opinion as then user need to know "how it would behave" when image is unset or null which is basically image icon not getting rendered so I thought to cut the chase and describe the final result

Reply:
> “will behave as if the image is null”

That's got the same mention of null that Joe was questioning

>"no exceptions will be thrown but no image icon will be rendered"

That seems OK to me. I'd just drop the word 'icon'.

Reply:
Updated the class-level javadoc